barustenberg/src/plonk/composer/standard_composer.rs

```diff
@@ -1103,7 +1103,7 @@ impl<RSF: ReferenceStringFactory> StandardComposer<RSF> {
     /// It first computes the verification key, then constructs a `Verifier`
     /// using the computed key and the manifest of public inputs.
     /// Finally, it adds a `KateCommitmentScheme` to the verifier and returns it.
-    fn create_verifier(&mut self) -> Result<Verifier<Keccak256, StandardSettings<Keccak256>>> {
```

Comment:
Why do we need to add this here

Reply:
oh no we removed it

Reply:
because this is specifically a plonk verifier